### ISSUE TITLE
post_meta don't display normally in the latest release [fix added]

### ISSUE THREAD

**Issue by dandyxu** [theme-next/hexo-theme-next#209]
### I agree and want to create new issue 

- [X] Yes, I was on [Hexo Docs page](https://hexo.io/docs/), especially on [Templates](https://hexo.io/docs/templates.html), [Variables](https://hexo.io/docs/variables.html), [Helpers](https://hexo.io/docs/helpers.html) and [Troubleshooting](https://hexo.io/docs/troubleshooting.html).
- [X] Yes, I was on [NexT Documentation Site](http://theme-next.org/docs/).
- [X] And yes, I already searched for current [issues](https://github.com/theme-next/hexo-theme-next/issues?utf8=%E2%9C%93&q=is%3Aissue) and this is not help to me.

***

### Expected behavior 
When I upgrade hexo to latest 3.7.0 and download latest release version of theme-next (v6.1.0), everything is fine but all post_meta don't display normally. For example, in the post_meta area, by default, it should display "Posted on 2018-04-03". 

### Actual behavior 
But actually it displays as "post.posted 2018-04-03". The same as in the footer: Powered -> footer.powered etc.

### Steps to reproduce the behavior 
1. Install hexo-cli and initial a new hexo from scratch;
2. Clone theme-next v6.1.0 to the themes folder;
3. Switch theme in the hexo _config.yml file to next and hexo generate;
4. Hexo server and you will be able to see the issue.

* Link to demo site with this issue: N/A
* Link(s) to source code or any usefull link(s): N/A

### Node.js and NPM Information

```
node: v9.3.0
npm: 5.6.0
```

### Package dependencies Information

```
all default
```

#### Hexo version

```
hexo: 3.7.0
hexo-cli: 1.1.0
```

#### Hexo Configuration

```yml
default yml file
```

### NexT Information

**NexT Version:**

- [ ] Latest Master branch
- [x] Latest Release version
- [ ] Old version

**NexT Scheme:**

- [x] All schemes
- [ ] Muse
- [ ] Mist
- [ ] Pisces
- [ ] Gemini

#### NexT Configuration:

```yml
all default
```

### Other Information 
https://imgur.com/a/yj3oo

**Comment by sli1989** [comment]
Live demo?

**Comment by ivan-nginx** [comment]
@sli1989 no need to reproduce this bug in other NexT version. I think it's appear after update to [Hexo v3.7.0](https://github.com/hexojs/hexo/releases/tag/3.7.0) with `[use default language](https://github.com/hexojs/hexo/pull/3069)` fixes. So, need just to reproduce this bug under this Hexo version and try to resolve this.

### Possible bugs:

* In Hexo's default config `language` option not setted by default, e.g. `language: `
* In Hexo's default config set `language: default`

### Possible fixes:

* Add warning notes to docs about setting `language` option in main Hexo's config
* Create `default.yml` symlink on `_en.yml`

***

@dandyxu can u confirm what in previous [Hexo v3.6.0](https://github.com/hexojs/hexo/releases/tag/3.6.0) this bug not detected?

**Comment by sli1989** [comment]
Need to set `language: lang` in  Hexo v3.7.0. Can't leave it blank.

**Comment by dandyxu** [comment]
@sli1989 @ivan-nginx Thanks for your responses and helps. As you said, the issue was happened because of `language` settings. Previously I set `language : en` and now I only need to change to `language : _en` or rename `_en.yml` file to `default.yml`, the issue is resolved.

@ivan-nginx I tried to install Hexo v3.6.0 but everytime I can only install the latest version v3.7.0. I'm sorry that I'm new to Hexo and I couldn't find any tutorials about install specific version of Hexo neither. So I probably can't confirm whether this bug is also appeared in Hexo v3.6.0. However, if you can direct me some tutorials, I'm very appreciated about that. Thank you.

@sli1989 Thank you very much as well for trying to figure this issue out for me.

**Comment by ivan-nginx** [comment]
@dandyxu @sli1989 fixed in #210 and merged into master branch.

For now u can defune language as u want (e.g. `language: en`, `language: default` or `language:`) — will be used `en` language as default.

If issue solved, close it please.
